src/wmtk/attribute/AttributeManager.hpp

```diff
@@ -137,6 +142,21 @@ MeshAttributeHandle<T> AttributeManager::register_attribute(
     return r;
 }
 template <typename T>
+inline T AttributeManager::parent_scope(std::function<T()> f)
+{
+    change_to_parent_scope();
```

Comment:
this change to parent scope needs to happen for all meshes simultaneously (in case the user wants to specify reading some previous state of another mesh in the system?). The change_to_parent_scope should probably invoke some functor like teh AttributeScopeHandle object did to switch all meshes to a parent scope or child scope